### PR TITLE
Hortense maintenance 20-26 April 2023

### PR DESCRIPTION
**Pull request by epauwels**

### CODE REVIEW

File: incidents/hortense_20230331.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Impacted: all login nodes, all compute nodes and the shared scratch filesystem will be offline.
+  
+  As a next step towards integrating the second phase of Tier1 Hortense, we have to schedule another maintenance window.
+  Please already take this into account when planning your further computational experiments.
+  The works involve a.o.:
+    - more extensive testing of phase2 hardware
+    - making sure that cooling systems can feed phase 1+2 in a stable fashion
+    - further configuration of the scratch storage system extension
+
+  Impacted: all login nodes, all compute nodes, and the shared scratch filesystem will be offline.
```
Comment (boegel) [inline incidents/hortense_20230331.yml:19]: add HTML tags to preserve formatting:

```suggestion
  <p>Repairs will be done to the existing nodes and preparatory work is planned to the datacenter and hardware in view of the second phase of Tier-1 Hortense.<br/>
  Impacted: all login nodes, all compute nodes and the shared scratch filesystem will be offline.</p>
  
  <p>As a next step towards integrating the second phase of Tier1 Hortense, we have to schedule another maintenance window.<br/>
  Please already take this into account when planning your further computational experiments.
  The works involve a.o.:
  <ul>
  <li>more extensive testing of phase2 hardware</li>
  <li>making sure that cooling systems can feed phase 1+2 in a stable fashion</li>
  <li>further configuration of the scratch storage system extension</li>
  </ul>
  </p>
  <p>Impacted: all login nodes, all compute nodes, and the shared scratch filesystem will be offline.</p>

**Review by boegel — CHANGES_REQUESTED** [review]

File: incidents/hortense_20230331.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+meta_data:
+  title: Scheduled maintenance Tier-1 Hortense
+  start_date: 2023-04-20 06:00:00
+  end_date: 2023-04-26 18:00:00
+  affected: tier1_compute
+  level: high
+  planned: yes
+content: >
+  <p>Repairs will be done to the existing nodes and preparatory work is planned to the datacenter and hardware in view of the second phase of Tier-1 Hortense.<br/>
```
Comment (itkovian) [inline incidents/hortense_20230331.yml:9]:
```suggestion
  <p>Repairs will be made to the existing nodes. Preparatory work at the datacenter is planned w.r.t. the second phase of Tier-1 Hortense.<br/>
```

File: incidents/hortense_20230331.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  Impacted: all login nodes, all compute nodes and the shared scratch filesystem will be offline.</p>
+  
+  <p>As a next step towards integrating the second phase of Tier1 Hortense, we have to schedule another maintenance window.<br/>
+  Please already take this into account when planning your further computational experiments.
```
Comment (itkovian) [inline incidents/hortense_20230331.yml:13]:
```suggestion
  Please already take this into account when planning your future computational experiments.
```

File: incidents/hortense_20230331.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  The works involve a.o.:
+  <ul>
+  <li>more extensive testing of phase2 hardware</li>
+  <li>making sure that cooling systems can feed phase 1+2 in a stable fashion</li>
```
Comment (itkovian) [inline incidents/hortense_20230331.yml:17]:
```suggestion
  <li>making sure that the cooling systems can feed both phases 1 and 2 in a stable fashion</li>
```

**Comment by boegel** [comment]
@itkovian You'll need to open a follow-up PR if those suggestions are important, sorry

**Pull request merged**